<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExQDateTime</class>
 <widget class="QWidget" name="ExQDateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExQDateTime</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>941</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>日期和时间(QTime、QDate、DateTime)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1" colspan="2">
        <widget class="QTimeEdit" name="timeEdit"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lEditDate"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>日期时间</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="lEditDateTime"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QPushButton" name="btnGetDateTime">
         <property name="text">
          <string>获取当前的时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>日期</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labCurrTime">
         <property name="text">
          <string>当前时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lEditTime"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>定时器</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labGo">
         <property name="text">
          <string>等待</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>定时周期</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnSetPeriod">
         <property name="text">
          <string>设置定时周期</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
       <item row="2" column="1">
        <widget class="QLCDNumber" name="lcdNumber_2"/>
       </item>
       <item row="2" column="2">
        <widget class="QLCDNumber" name="lcdNumber_3"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>日历组件(Calendar Widget)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>选择的日期</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lEditCalendar"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCalendarWidget" name="calendarWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
